### DISCUSSION TITLE
请问如何如何在自定义词典｢custom_phrase.txt｣里限制词汇?

### DISCUSSION THREAD

**Discussion by nutnutwin** [rime/home#924] [Q&A] [answered]
比如在输入abc后, 不要显示｢按补充｣
能否用custom_phrase.txt实现? 谢谢

**Comment by LEOYoon-Tsaw** [comment]
沒看懂

**Comment by nutnutwin** [comment]
> 沒看懂

你好, 不好意思没有解释清楚
比如我在custom_phrase.txt里有下列定义
输入｢t｣可以得到｢! ｣, 输入｢bfh｣可以得到｢%｣
```
! 	t	255
%	bfh	255
```
--- ✄ -----------------------
现在我输入｢abc｣得到了｢按补充｣(应该是默认变换出来的)
请问能否修改custom_phrase.txt来阻止某个特定词组出现? 

谢谢
--- ✄ -----------------------
类似的在Google 日语输入法里

```
a	b	抑制単語
```
可以让输入｢a｣时候无法变换成｢b｣

**Comment by Endif12** [comment]
这个需求还蛮少见的

**Comment by LEOYoon-Tsaw** [comment]
弄個不可見字符試試？還是會有選項，但可以顯示成空的

**Comment by nutnutwin** [comment]
> 弄個不可見字符試試？還是會有選項，但可以顯示成空的

你好, 谢谢你的建议. 

我希望能屏蔽某些词汇以防打错...
比如在公司环境下屏蔽不适宜的词

所以定义成空格应该不行. 还是谢谢你

**Comment by lotem** [comment, marked as answer]
關閉自動造句選項，可以減少無意義的詞組。（同時也減少有意義的詞組，習慣打字詞不打長句可以考慮）
以`luna_pinyin`方案爲例：
```yaml
# luna_pinyin.custom.yaml
patch:
  translator/enable_sentence: false
```

另外，只要在`custom_phrase.txt`任意設置一個輸入碼爲`abc`的條目，例如文字隨意設爲`abc`，他會成爲該輸入的默認候選項，而通過拼音計算的詞組排在後邊，也可以防打錯。

**Comment by nutnutwin** [comment]
@lotem 感谢回答
